src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/data/model/Client.kt

```diff
@@ -50,12 +54,21 @@ data class Client(
   @Column(length = 2000)
   @Convert(converter = TokenSettingsConverter::class)
   var tokenSettings: TokenSettings,
+
+  @OneToMany
+  @JoinColumn(name = "registeredClientId")
+  @Where(clause = "access_token_issued_at =(select max(oa.access_token_issued_at) from oauth2_authorization oa)")
+  val authorization: MutableSet<Authorization> ? = mutableSetOf(),
```

Comment:
Rather than just `authorization`, would be better to call this field something more meaningful, like maybe: `mostRecentClientCredentialsToken`

Reply:
Updated